CHANGELOG.md

```diff
@@ -19,6 +19,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 - `tnt_clock_delta` metric to compute clock difference on instances
 - set custom global labels in config and with `set_labels` function [#259](https://github.com/tarantool/metrics/issues/259)
+- set custom handlers with `set_custom_handlers` function [#120](https://github.com/tarantool/metrics/issues/120),
+  [#242](https://github.com/tarantool/metrics/issues/242)
```

Comment:
How's #242 related to this patch?

Reply:
#120 and #242 is about a same thing - custom handlers. Users could customize health response with custom health handler